tests/test_custom_models.py

```diff
@@ -1072,8 +1124,8 @@ def test_disable_adapters_with_merging(self, test_name, model_id, config_cls, co
         # unmerged or merged should make no difference
         assert torch.allclose(outputs_after, outputs_unmerged, atol=atol, rtol=rtol)
 
-        # check that disabling adapters gives the same results as before training
-        assert torch.allclose(outputs_before, outputs_disabled, atol=atol, rtol=rtol)
+        # check that disabling adapters gives the same results as base model
+        assert torch.allclose(outputs_base, outputs_disabled, atol=atol, rtol=rtol)
```

Comment:
If `init_weights` is implemented for VB-LoRA, this can be reverted, right?

Reply:
Yes, but we are having difficulty implementing `init_weights` in the same way as LoRA.

Reply:
Updated.